sklearn/decomposition/_kernel_pca.py

```diff
@@ -44,19 +45,20 @@ class KernelPCA(TransformerMixin, BaseEstimator):
         Independent term in poly and sigmoid kernels.
         Ignored by other kernels.
 
-    kernel_params : mapping of string to any, default=None
-        Parameters (keyword arguments) and values for kernel passed as
-        callable object. Ignored by other kernels.
+    kernel_params : dict, default=None
+        Mapping of string to any. Parameters (keyword arguments) and
```

Comment:
Nit: indicating that these are keyword arguments is enough to disambiguate the type of the param

```suggestion
        Parameters (keyword arguments) and
```